Update PDF quality and update cyclodextrin derivatives README.md

bergazin opened the pull request:
@davidlmobley Included higher quality PDFs of the sampl7 host structures. One includes the names under each structure, the other doesn't. Updated the cyclodextrin derivatives README.md since we’re working with 8 cyclodextrin derivatives instead of 9. So we have a total of 9 hosts, the 8 CD's plus beta-cyclodextrin.
[sampl_host_structures_coded.pdf](https://github.com/MobleyLab/SAMPL7/files/3208884/sampl_host_structures_coded.pdf)
[sampl_host_structures_1.pdf](https://github.com/MobleyLab/SAMPL7/files/3208885/sampl_host_structures_1.pdf)

bergazin commented:
new jpgs made from the updated pdfs
![sampl_host_structures_1](https://user-images.githubusercontent.com/29715177/58231210-49461800-7ceb-11e9-9102-6bcb47ea8a80.jpg)
![sampl_host_structures_coded](https://user-images.githubusercontent.com/29715177/58231211-49461800-7ceb-11e9-9b30-1a675ba90f5f.jpg)

davidlmobley commented on host_guest/cyclodextrin_derivatives/README.md:
```diff
@@ -1,6 +1,6 @@
-# The SAMPL7 Cyclodextrin Derivatives Challenge
+README.md# The SAMPL7 Cyclodextrin Derivatives Challenge
```
Looks like you acidentally inserted a "README.md" remark where it doesn't belong?

bergazin replied:
I think I was copying/pasting info from the readme file and accidently inserted "README.md". Strange. Fixed now!

bergazin commented:
I added added a short description to the readme file in the cyclodextrin_derivatives directory about the images directory, added some descriptions to the readme file in cyclodextrin_derivatives/hosts, and added a readme to cyclodextrin_derivatives/images that lists some descriptions

davidlmobley commented on host_guest/cyclodextrin_derivatives/hosts/README.md:
```diff
@@ -1,3 +1,9 @@
 
-Provided are the hosts for this challenge in PDB and ChemDraw format, including beta cyclodextrin as well as the series of modified cyclodextrins.
+Provided are the hosts for this challenge in PDB and ChemDraw (.cdx) format, including beta cyclodextrin as well as the series of modified cyclodextrins.
+`MGLab8.pdb, MGLab9.pdb, MGLab23.pdb, MGLab24.pdb, MGLab34.pdb, MGLab35.pdb, MGLab36.pdb` : PDB host files provided by Katy Kellett April 26th, 2019.
```
In future commits, will look nicer if you put each in single quotes. But this will work for now.

davidlmobley commented on host_guest/cyclodextrin_derivatives/hosts/README.md:
```diff
@@ -1,3 +1,9 @@
+`MGLab8.pdb, MGLab9.pdb, MGLab23.pdb, MGLab24.pdb, MGLab34.pdb, MGLab35.pdb, MGLab36.pdb` : PDB host files provided by Katy Kellett April 26th, 2019.
+`b-CD.pdb` : PDB file of host beta-cyclodextrin provided by Katy Kellett April 26th, 2019.
+`MGLab_8.cdx, MGLab_9.cdx, MGLab_19.cdx, MGLab_23.cdx, MGLab_24.cdx, MGLab_29.cdx, MGLab_34.cdx, MGLab_35.cdx, MGLab_36.cdx` : ChemDraw files of the modified cyclodextrin hosts provided by Katy Kellett May 10th, 2019.
+`bCD.cdx` : ChemDraw file of host beta-cyclodextrin provided by Katy Kellett May 10th, 2019.
```
Try to name consistently; if it's `b-CD.pdb` above it should be `bCD.cdx` here.
